content/installation/linux/LinuxRepoConfig.md

```diff
@@ -0,0 +1,66 @@
+
+<!--
+title: "Linux Package Distribution"
+description: "How to use Contrast's Linux repository"
+tags: "installation linux package repo distribution"
+-->
+
+## About Linux Package Distribution
+
+Some of Contrast's products are distributed through Linux packages. This makes installation and updates very easy and familiar as it utilizes the Linux packaging and distribution system. 
+
+To use the Contrast package repository, you must configure your package management system to include the appropriate reference.
+
+While Contrast packages are tied to the specific Linux distribution you're using, some are applicable to any distribution. The following steps for **Ubuntu** and **RedHat/CentOS** users configure both distribution points; once configured, you will have access to all our Linux-packaged products.
+
+**This is a one-time configuration step.** Once its done, you're connected to the Contrast software distribution pipeline, and can solely manage it through your distribution's package manager. 
+
+> **Note:** Some Linux package managers need a public GPG key that's used to verify the signed packages from our repository; this is included in the configuration steps, if its applicable to your system.
```

Comment:
it's

Reply:
Yes. I fixed the wrong typo in my latest commit.